infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/KubernetesCheApiInternalEnvVarProvider.java

```diff
@@ -28,7 +28,8 @@
   private final String cheServerEndpoint;
 
   @Inject
-  public KubernetesCheApiInternalEnvVarProvider(@Named("che.api") String cheServerEndpoint) {
+  public KubernetesCheApiInternalEnvVarProvider(
```

Comment:
Are there any documentation changes that are needed for this change?
Do we need to propagate external API URL for containers?

Reply:
Looks like there are no container which used external API url.